ads_plugin.py

```diff
@@ -601,7 +603,7 @@ def on_compute_morphometrics_button(self, event):
 
         # Compute statistics
         stats_array = get_axon_morphometrics(im_axon=pred_axon, im_myelin=pred_myelin, pixel_size=pixel_size)
-
+        # print(stats_array)
```

Comment:
Is this intentional to have the print here, or was it used for debug and should be removed?

Reply:
It was used for debugging, thanks for catching it. I fixed it.